docs/emulator-suite/_includes/_firestore_admin_quickstart.md

```diff
@@ -0,0 +1,6 @@
+The Firebase Admin SDKs automatically connect to the Cloud Firestore
```

Comment:
Extracted into its own file. We can modify devsite to include this file as well. No actual changes.